### ISSUE TITLE
Can slf4j-simple dependency be removed?

### ISSUE THREAD

**Issue by hanslovsky** [scenerygraphics/scenery#99]
I have a project that transitively depends on `scenery`, which introduces `slf4j-simple` as a dependency:
```
mvn dependency:tree
[INFO] +- sc.fiji:bigcat:jar:0.0.3-SNAPSHOT:compile
[INFO] |  +- net.sf.trove4j:trove4j:jar:3.0.3:compile
[INFO] |  +- net.imagej:ij:jar:1.51n:compile
[INFO] |  +- com.beust:jcommander:jar:1.48:compile
[INFO] |  +- org.zeromq:jzmq:jar:3.1.0:compile
[INFO] |  +- org.scijava:script-editor:jar:0.1.4:compile
[INFO] |  |  +- org.scijava:scripting-java:jar:0.4.1:compile
[INFO] |  |  |  \- org.scijava:minimaven:jar:2.2.1:compile
[INFO] |  |  +- com.fifesoft:rsyntaxtextarea:jar:2.6.1:compile
[INFO] |  |  +- com.fifesoft:languagesupport:jar:2.6.0:compile
[INFO] |  |  |  +- com.fifesoft:autocomplete:jar:2.6.1:compile
[INFO] |  |  |  \- org.mozilla:rhino:jar:1.7.6:compile
[INFO] |  |  \- com.miglayout:miglayout:jar:swing:3.7.4:compile
[INFO] |  \- graphics.scenery:scenery:jar:0.1.1-SNAPSHOT:compile
[INFO] |     +- org.slf4j:slf4j-simple:jar:1.7.25:compile
```

This causes conflicts with `Apache Spark`, which also introduces a `slf4j` binding:
```
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/hanslovskyp/.m2/repository/org/slf4j/slf4j-simple/1.7.25/slf4j-simple-1.7.25.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/hanslovskyp/.m2/repository/org/slf4j/slf4j-log4j12/1.7.16/slf4j-log4j12-1.7.16.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.SimpleLoggerFactory]
```
This makes it impossible for me to disable `Spark` logging, which clutters up the console in local mode and makes debugging virtually impossible.

I can get around this by adding
```xml
<exclusions>
	<exclusion>
		<groupId>org.slf4j</groupId>
		<artifactId>slf4j-simple</artifactId>
	</exclusion>
</exclusions>
```
to the dependency that introduces `scenery`/`slf4j-simple`. It would, however, be preferable to not include `slf4j-simple` as a dependency at all. Unfortunately, I am not familiar with the contents of scenery and I don't know if it ships with an executable, which would require including `slf4j` binding such as `slf4j-simple` as dependency. If executables only exist as tests, it would suffice to change the scope of the `slf4j-simple` dependency to `test`.

**Comment by skalarproduktraum** [comment]
Hi @hanslovsky!

Apologies for the delay, just arrived back from vacation.
You are right, as scenery is intended as a library, and actually only ships tests (which then of course need slf4j-something), so slf4j-simple should be removed. I'll address this later this week and update this issue accordingly! 

cheers!

**Comment by hanslovsky** [comment]
Probably just setting the scope property to test is the best way to go here, no?

**Comment by skalarproduktraum** [comment]
Exactly! Just didn't have time to do that, was still catching up with other things. Good news now though, fixed in 0eb13c250aeeace1196af9de3461cb169d5f918b 👍 
Thanks for spotting this!
